distributed/cli/tests/test_dask_ssh.py

```diff
@@ -1,9 +1,24 @@
 from click.testing import CliRunner
 
 from distributed.cli.dask_ssh import main
+from distributed.utils_test import popen
 
 
 def test_version_option():
     runner = CliRunner()
     result = runner.invoke(main, ["--version"])
     assert result.exit_code == 0
+
+
+def test_nprocs_renamed_to_num_workers():
+    with popen(["dask-ssh", "--nprocs=2", "127.0.0.1"]) as cluster:
+        assert any(
+            b"renamed to --num-workers" in cluster.stderr.readline() for i in range(15)
+        )
```

Comment:
This test is timing out on Python 3.7 and 3.8, but curiously not on 3.9. I've set up a Docker container to try and debug it locally.